src/components/attendee/SpaceInfoText/SpaceInfoText.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import ShowMoreText from "react-show-more-text";
+import cn from "classnames";
+
+import { AnyVenue } from "types/venues";
+
+import { RenderMarkdown } from "components/organisms/RenderMarkdown";
+
+import CN from "./SpaceInfoText.module.scss";
+
+const descriptionLines: Record<SpaceInfoTextVariant, number> = {
+  space: 1,
+  mobilePortal: 5,
+  desktopPortal: 5,
+};
```

Comment:
[Question] Are these values documented somewhere or can we export them from some settings file with comments to what they mean?

Reply:
no, it's not documented. It's the number of lines showed by default if the text is long. If it's longer than xx lines than it's expandable.

I chose the default number of lines for description. Later we'll limit it in admin by character limit.
![image](https://user-images.githubusercontent.com/63313732/166433908-76a313bb-7833-48d2-9f67-bc573bcc97c2.png)

and in space (according to the design) there's just 1 line showed by default.
<img width="430" alt="image" src="https://user-images.githubusercontent.com/63313732/166436684-305f7e15-9c26-49ae-883b-b842e7d5e4b1.png">